services/database/serverless.yml

```diff
@@ -242,7 +242,7 @@ resources:
     BannerTableStreamArn:
       Value: !GetAtt BannerTable.StreamArn
     FormTemplateTableName:
```

Comment:
would this need to change too?

Reply:
The output name doesn't need to change, just the ref